openmc/openmoc_compatible.py

```diff
@@ -0,0 +1,747 @@
+import copy
+import operator
+
+import numpy as np
+
+try:
+    import openmoc
+except ImportError:
+    raise ImportError('Unable to import openmoc which is needed by '
+                      'openmc.openmoc_compatible')
```

Comment:
I agree - I thought I recalled having this very same conversation a year ago or so, and was actually surprised to find this `try-except` block here yesterday. I'll remove it now.